packages/net/tcp_connection.pony

```diff
@@ -758,12 +759,12 @@ actor TCPConnection
         _next_size = _max_size.min(_next_size * 2)
       end
 
```

Comment:
this change is non-windows only, correct?

Reply:
Correct, there is no logical change to the windows code. The only change to the windows code is to migrate from `_read_len` to `_read_buf_offset`.